app/app.js

```diff
@@ -1,10 +1,22 @@
 const express = require('express')
 const app = express()
-const port = 3000;
+const port = process.env.PORT;
 
 app.listen(port);
-console.log(`Server running at http://localhost: ${port}`);
+console.log(`Server running at http://${process.env.IP}:${port}`);
+
 app.get('/', (req, res) => {
   const name = process.env.NAME || 'unknown';
   res.send(`Hello ${name}!`);
-});
\ No newline at end of file
+});
+
+// Ensure that NAME is set in the environment. If it is not, then the health
+// check should fail.
+app.get('/healthz', (req, res) => {
```

Comment:
This endpoint should be improved on over time as the responsibilities of the app changes. The Kubernetes liveness and readiness probes will use this to check the health of the application.